lib/openc_bot/tasks.rb

```diff
@@ -16,6 +18,23 @@ def pid_dir
   end
 end
 
+task "resque:setup" do
+  bot_name = get_bot_name
+  require_relative File.join(Dir.pwd, "lib", bot_name)
+
+  Resque.logger = Logger.new(ENV.fetch("EXTERNAL_BOTS_LOG", $stdout))
+  Resque.logger.level = ENV.fetch("EXTERNAL_BOTS_LOG_LEVEL", Logger::DEBUG)
+  Resque.logger.info "Resque setup"
+
+  resque_config_file = ENV.fetch("EXTERNAL_BOTS_RESQUE_CONFIG", File.join(Dir.pwd, "..", "config", "resque.yml"))
```

Comment:
When I tested this out, I used a big command like this: `bundle exec openc_bot rake resque:work QUEUE=sru_request_cy VERBOSE=true FETCHER_BOT_ENV=production EXTERNAL_BOTS_RESQUE_CONFIG=/oc/openc/secure_config/openc/config/resque.yml`

So that's passing in a EXTERNAL_BOTS_RESQUE_CONFIG with a path to a suitable file (hotwiring it to go look at openc's configuration). We might do something similar when setting up workers in kubernetes. But I've put a default path there as `"../config/resque.yml"` which will (when the gem is used within a production bot) point to the higher config directory within the external_bots repo deployed on backgrounder2. So I imagine we might put that file in place once, and all bots will pick up that same resque.yml config. A tidy way of automatically putting that in place might involve some capistrano setting symlinks. That'll be a PR on the external_bots repo though, so we can work that out later.